x/gov/keeper/grpc_query.go

```diff
@@ -87,6 +87,12 @@ func (q Keeper) Proposals(c context.Context, req *v1.QueryProposalsRequest) (*v1
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	// Populate contents given that we are iterating the store keys and not using
```

Comment:
I wonder. If this is used by a CW contract. Maybe we extend the request type to include a field `include_content`? This way a client can indicate if they want content or not to save gas costs.